.github/workflows/developer-site-ci.yml

```diff
@@ -0,0 +1,31 @@
+name: Libra Developers Website CI
+
+on:
+  push:
+    branches: [ auto ]
+    paths: [ 'developers.libra.org/**' ]
+  pull_request:
+    branches: [ master ]
+    paths: [ 'developers.libra.org/**' ]
```

Comment:
We should remove the paths since we want to test rustdoc generation on rust code changes right?

Reply:
Yeah, I think you are right. Let's remove to see if it has the intended behavior.